pyproject.toml

```diff
@@ -53,7 +53,12 @@ dev = [
     "pre-commit",
     "setuptools_scm",
 ]
-napari = ["napari[pyside2]", "brainglobe-napari-io", "cellfinder-napari<1.0.0"]
+napari = [
+    "napari[pyqt5]",
+    "brainglobe-napari-io",
+    "cellfinder-napari<1.0.0",
+    "qtpy",
```

Comment:
```suggestion
```
Don't think we need this, because napari already [depends on `qtpy>=1.10.0`](https://github.com/napari/napari/blob/0312cacdb231687e6d6474d70b3eb2097c12ff1d/setup.cfg#L71)